src/content/RepoPage/RepoPage.js

```diff
@@ -28,45 +65,81 @@ const headers = [
   },
 ];
 
-const rows = [
-  {
-    id: '1',
-    name: 'Repo 1',
-    createdAt: 'Date',
-    updatedAt: 'Date',
-    issueCount: '123',
-    stars: '456',
-    links: 'Links',
-  },
-  {
-    id: '2',
-    name: 'Repo 2',
-    createdAt: 'Date',
-    updatedAt: 'Date',
-    issueCount: '123',
-    stars: '456',
-    links: 'Links',
-  },
-  {
-    id: '3',
-    name: 'Repo 3',
-    createdAt: 'Date',
-    updatedAt: 'Date',
-    issueCount: '123',
-    stars: '456',
-    links: 'Links',
-  },
-];
-
 const RepoPage = () => {
+  const LinkList = ({ url, homepageUrl }) => (
```

Comment:
`LinkList` and `getRowItems` should be declared outside the `RepoPage` const declaration. This is making your formatting off.

Reply:
Fixed https://github.com/carbon-design-system/carbon-tutorial/pull/901/commits/67cfb920cd743471a80dc6ec7c74c150fc887e22